docs/examples/manifests/user_monitoring/1_enable_user_workload_monitoring.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
```

Comment:
the names are correct? file name and object name seem different for me

Reply:
Yes it's correct. You are enabling user workload by configuring cluster monitoring:
```
  config.yaml: |
    enableUserWorkload: true
```

Reply:
Using rosa the config-map already exists, so far seems ok.  Perhaps have the customer check first before attempting to create? meh? 
preexisting config.
```
apiVersion: v1
data:
  config.yaml: |
    enableUserWorkload: true
```

Reply:
@weshayutin I have updated the documentation to edit certain ConfigMaps, so the already existing configurations are not overriden and also added note in which we warn use if the sample configuration is being used:

```
> **Note:** The configuration files that allows to enable User Workload Monitoring can be found in the [docs/examples/manifests/user_monitoring/](./examples/manifests/user_monitoring/) folder of this repository, however they will override any previous configuration that could have been on the cluster, so use them with caution. They are named with the number prefix, which represents the order in which they should be applied.
```